bokehjs/src/coffee/document.coffee

```diff
@@ -102,21 +102,24 @@ class Document
   solver: () ->
     @_solver
 
-  resize: (width=null, height=null) ->
+  resize: (jquery_event, width=null, height=null) ->
```

Comment:
if anybody has feedback on the advisability of the `jquery_event` placeholder arg, i'd appreciate it. It gets passed by JQuery.proxy and I don't know how to get around it.